### DISCUSSION TITLE
Incremental Static Generation and other advanced modes are now possible.

### DISCUSSION THREAD

**Discussion by n4an** [nuxt/nuxt#9929] [General]
What modes?

Can I generate a site and when I start it the api server run like in dev mode to send to client generated page and after make api call to update info

**Comment by kissu** [comment]
Hi, can you be a bit more explicit here please? 😌

**Reply by n4an** [reply]
when we generate a site with npm run generate and after run it with npm run start it simply send html.
if I open an url from server middleware got error, but it will be great to get api response, to update data on client side without regenerate, it can be a new special hook or some how use existent one.
Not full static

**Reply by kissu** [reply]
This may help: https://github.com/nuxt/framework/discussions/560
